### DISCUSSION TITLE
PyArmor 8 and 7 licenses.

### DISCUSSION THREAD

**Discussion by HangingWithCMM** [dashingsoft/pyarmor#1646] [Q&A] [answered]
Is it possible to have a PyArmor 7 license and PyArmor 8 license active at the same time? I want to use “Pyarmor old” and “Pyarmor Group” licenses at the same time, on the same device.

**Comment by jondy** [comment, marked as answer]
No problem. Please use option `--home` to specify another path, more information refer to man page about usage of option `--home`
